src/ape/types/signatures.py

```diff
@@ -84,6 +85,29 @@ class _Signature:
     The signature proof point (``s``) in an ECDSA signature.
     """
 
+    @field_validator("v", mode="before")
```

Comment:
you can use the `HexInt` type, it should do all this automatically when using that type